packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.base.tsx

```diff
@@ -97,11 +97,11 @@ export class DetailsHeaderBase extends BaseComponent<IDetailsHeaderBaseProps, ID
     this._onRootRef = this._onRootRef.bind(this);
     this._isEventOnHeader = this._isEventOnHeader.bind(this);
     this._onDropIndexInfo = {
-      sourceIndex: Number.MIN_SAFE_INTEGER,
-      targetIndex: Number.MIN_SAFE_INTEGER
+      sourceIndex: -1,

```

Comment:
I read through the code and it seems that -1 will achieve the same thing as MIN_SAFE_INTEGER. @laxmikankanala, looks like you added this code in #4857 -- can you confirm that -1 is safe? (or @aditima  or @yiminwu who reviewed)

Reply:
Code seems fine to me. As long as column reordering works, particularly at the edges of the table then I think we're OK.

Reply:
@KevinTCoughlin Would you mind helping test that? (Guessing you're more familiar with the intended behavior.)

Reply:
Works well in my testing

Reply:
Thanks for checking!